app/views/members/members/show.html.haml

```diff
@@ -0,0 +1,118 @@
+- content_for(:page, 'member')
+- content_for(:title, @member.name)
+
+
+.row
+  .col-lg-4
+    .card
+      .card-body.member-card
+        %h2
+          %i.fas.fa-user-circle
+          = @member.name
+
+        - if @member.tags.any?
+          %ul
+            - @member.tags.each do |tag|
+              %li= I18n.t(tag.name, scope: 'activerecord.attributes.tag.names')
+
+        %p
+          %i.fa.fa-map-marker
+          = @member.address
+          = @member.house_number
+          %br
+          = @member.postal_code
+          = @member.city
+        %p
+          %i.fa.fa-phone-alt
+          = @member.phone_number
+        %p
+          %i.far.fa-envelope-open
+          %a{:href => "mailto:#{@member.email}"}= @member.email
+
+        .btn-group
+          .row
+            .col-md-6
+              = link_to :edit_current_member, class: 'btn btn-primary btn-sm btn-block' do
+                %i.far.fa-edit
+                Bewerken
+            .col-md-6
+              = link_to :password_current_member, { :class => 'button btn btn-sm btn-default btn-block' } do
+                %i.fa.fa-key
+                = I18n.t 'members.home.edit.password_options.change_password'
+
+    .card
```

Comment:
The second dropdown is a bit too large for the card

Reply:
which dropdown do you main, as far as I can see there is only one dropdown in members/members/show